### ISSUE TITLE
Question/Issue with LIKE operators

### ISSUE THREAD

**Issue by borisrorsvort** [stefankroes/ancestry#504]
Not so familiar with sql operators but when running the descendants, I get this error (.children runs fine though). 

<img width="1348" alt="CleanShot 2020-08-25 at 09 39 48@2x" src="https://user-images.githubusercontent.com/39081/91146275-f362db80-e6b6-11ea-85f6-5e3971bc1057.png">

Can someone help?

**Comment by kbrock** [comment]
Hello @borisrorsvort 

This seems to work for other people. So chances are it is a problem with how you created the table, defined ancestry, or are calling it.

Please share the schema (or at least the part that defines post_records)
Also share the model (or at least the part that defines the has_ancestry

It almost looks like you defined your ancestry field as `post_record_id`

from there we can hopefully get you all set

**Comment by borisrorsvort** [comment]
The part regarding ancestry is this:

```ruby
create_table "post_records", id: :serial, force: :cascade do |t|
  t.bigint "post_record_id"
  t.string "ancestry"
end
```

```ruby
has_many :post_records, dependent: :destroy
has_ancestry ancestry_column: 'post_record_id'
```

**Comment by borisrorsvort** [comment]
Thanks! I realised I should have the read the readme more thoroughly. I 🤦  after reading :
> It almost looks like you defined your ancestry field as post_record_id

Now isn’t there to auto assign parent when you build associations like `@post_record.post_records.create(...)` would auto assign post_record_id?

**Comment by kbrock** [comment]
@borisrorsvort for batch loading a big tree or for loading a nested tree, the story for populating the `parent_id` is very lacking.

I did come up with better ways for populating a bunch of records based upon parent_id, but this is not totally applicable for most of our use cases.

I'll revisit and see if I can come up with a better method. but that unfortunately is probably not within your timeframe. I'm assuming that if it were simple, it would have been implemented by now.
